ligo/src/proposal.mligo

```diff
@@ -349,3 +349,5 @@ let drop_proposal (proposal_key, config, store : proposal_key * config * storage
     ([%Michelson ({| { FAILWITH } |} : string * unit -> return)]
       ("FAIL_DROP_PROPOSAL_NOT_OVER", ()) : return)
 
+let flush(param, config, store : nat * config * storage) : return =
+  not_implemented("flush")
```

Comment:
This shouldn't be here, probably from rebase.

Reply:
Removed.